### ISSUE TITLE
Documentation: Add configuration instructions

### ISSUE THREAD

**Issue by nhooey** [outr/scarango#70]
The documentation could use some configuration instructions, like defining the host and port of the ArangoDB instance.

Having configuration for timeouts would be great as well, including defaults.

**Comment by darkfrog26** [comment]
I agree this should be documented better, but this is already configurable: https://github.com/outr/scarango/blob/master/driver/src/main/scala/com/outr/arango/core/ArangoDBConfig.scala#L10

When you instantiate the `Graph` you can optionally supply your own `ArangoDBConfig`, or you can just define those configuration items via Profig. In the latter case, you can define a `config.json` file like:
```json
{
  "arangodb": {
    "username": "root",
    "password": "password"
  }
}
```

Or you can specify via environment variables:
`ARANGODB_USERNAME=root`
`ARANGODB_PASSWORD=password`
Or you can pass in command-line arguments:
`myApp -arangodb.username=root -arangodb.password=password`

If you're up to creating a PR to add this to the README that would certainly be welcome.

**Comment by darkfrog26** [comment]
@nhooey I'm closing this for lack of response. Please open a new ticket if you need further assistance.

**Comment by nhooey** [comment]
The documentation still doesn't have configuration instructions.
I'm not sure why it makes sense to close the ticket because I didn't respond.

**Comment by darkfrog26** [comment]
That's fair.
